service/src/main/java/com/commercetools/pspadapter/payone/mapping/BankTransferInAdvanceRequestFactory.java

```diff
@@ -31,9 +36,19 @@ public BankTransferInAdvanceRequest createPreauthorizationRequest(@Nonnull Payme
         return createRequestInternal(RequestType.PREAUTHORIZATION, paymentWithCartLike, BankTransferInAdvanceRequest::new);
     }
 
+    /**
+     * <b>NOTE:</b> this is potentially dangerous transaction type for this payment method. See more in
+     * {@link com.commercetools.pspadapter.payone.transaction.paymentinadvance.BankTransferInAdvanceChargeTransactionExecutor BankTransferInAdvanceChargeTransactionExecutor}
+     * @see com.commercetools.pspadapter.payone.transaction.paymentinadvance.BankTransferInAdvanceChargeTransactionExecutor
+     */
     @Nonnull
     @Override
     public BankTransferInAdvanceRequest createAuthorizationRequest(@Nonnull PaymentWithCartLike paymentWithCartLike) {
+        LOG.warn("Unsupported transaction type \"{}\" for payment method \"\".\n" +
+                        "\t\tNote: this payment/transaction type officially not supported by Payone " +
```

Comment:
`this payment/transaction type **is** officially`

Reply:
5eda8812